### DISCUSSION TITLE
couldn't been able to switch to supported chain if my wallet is connected to unsupported chain

### DISCUSSION THREAD

**Discussion by ethdev279** [WalletConnect/walletconnect-monorepo#3745] [Developer Support]
Noticed in v3 that if my wallet is connected to unsupported chain, I couldn't been able to switch to supported chain from acccount view. I was able to switch in v2 but not in v3. this is happening only when I have single chain in supported chains list in wagmi config.

![Screenshot 2023-10-06 163543](https://github.com/WalletConnect/walletconnect-monorepo/assets/45661693/1b374755-5b46-4cc0-af30-7bd956205c48)

**Comment by xzilja** [comment]
closing in favor of https://github.com/WalletConnect/web3modal/issues/1377 this is an actual bug / edge case that we will fix 👍
